app/gui/view/component-browser/component-list-panel/grid/src/entry/style.rs

```diff
@@ -5,63 +5,137 @@ use crate::prelude::*;
 use enso_frp as frp;
 use ensogl_core::data::color;
 use ensogl_core::display::shape::StyleWatchFrp;
-use ensogl_core::Animation;
+use ensogl_core::display::style::data::DataMatch;
 use ensogl_derive_theme::FromTheme;
 use ensogl_hardcoded_theme::application::component_browser::component_list_panel::grid as grid_theme;
 use entry_theme::highlight::selection as selection_theme;
 use grid_theme::entry as entry_theme;
 
 
 
+/// Color of the different parts of the entry.
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum Color {
+    /// The final color is a result of mixing the application background color with the "main"
+    /// color of the component group using a given coefficient.
+    MainColorIntensity(f32),
+    /// The final color is defined as an arbitrary color in the stylesheet.
+    Arbitrary(color::Lcha),
+}
+
+impl Default for Color {
+    fn default() -> Self {
+        Self::MainColorIntensity(0.0)
```

Comment:
does it mean that its fully transparent? If so, the can be accidentally surprising.

Reply:
Changed it to 1.0